data/src/test/java/org/apache/iceberg/data/TestDataFileIndexStatsFilters.java

```diff
@@ -217,10 +214,11 @@ public void testEqualityDeletePlanningStatsFilter() throws IOException {
       tasks = Lists.newArrayList(tasksIterable);
     }
 
-    Assert.assertEquals("Should produce one task", 1, tasks.size());
+    Assertions.assertThat(tasks.size()).isEqualTo(1).as("Should produce one task");
```

Comment:
we typically want to use `assertThat(tasks).hasSize(x)` as that will show the content of the collection if the assertion ever fails. Using `assertThat(tasks.size()).isEqualTo(x)` will only say `x != size` but you won't know the content